README.md

```diff
@@ -85,7 +85,7 @@ $ litmusctl version
 To connect Litmus Chaos agent:
 
 ```shell
-$ litmusctl agent connect
+$ litmusctl agent agent
```

Comment:
Should this be `litmusctl create agent` as I understand from the new dir structure

Reply:
Yes, @arkajyotiMukherjee . We need to change it once we released it.

Reply:
is `litmusctl agent agent` a valid command even for the old release.
If we mean to change this after the release then maybe we should revert the change to `litmusctl agent connect` @rajdas98

Reply:
Okay, it's a typo. Sorry

Reply:
Done @arkajyotiMukherjee